webhook_server/tests/test_github_api.py

```
@@ -1640,3 +1650,63 @@ def rmtree_fail(*args, **kwargs):
                                         await gh.cleanup()
 
                                         mock_logger.warning.assert_called()
+
+    def test_validate_custom_check_runs_builtin_collision(
+        self, minimal_hook_data: dict, minimal_headers: dict, logger: Mock
```

Ruff (0.14.10)

Unused method argument: `logger`

(ARG002)